tests/pfcwd/test_pfcwd_function.py

```diff
@@ -422,11 +422,19 @@ def storm_setup(self, init=False, detect=True):
                          }
             self.peer_dev_list[self.peer_device] = peer_info['hwsku']
 
+            if self.dut.topo_type == 't2':
+                gen_file = 'pfc_gen_t2.py'
+                pfc_send_time = 60
+            else:
+                gen_file = 'pfc_gen.py'
+
             # get pfc storm handle
             if init and detect:
                 self.storm_hndle = PFCStorm(self.dut, self.fanout_info, self.fanout,
                                             pfc_queue_idx=self.pfc_wd['queue_index'],
                                             pfc_frames_number=self.pfc_wd['frames_number'],
+                                            pfc_send_period=pfc_send_time,
```

Comment:
pfc_send_time is not defined for non-t2 topologies

Reply:
During PFCStorm class initialization, '_pfc_send_period_' definition has already been taken care even for non-t2 topologies as shown below. 
https://github.com/sanjair-git/sonic-mgmt/blob/30b05c0af3d533868c70520da922e30256719c16/tests/common/helpers/pfc_storm.py#L45

`self.pfc_send_period = kwargs.pop('pfc_send_period', None)`

Reply:
pfc_send_time still has to be defined otherwise it will result in a NameError. Has this code been tested on a non-t2 topology?

Reply:
Defined _pfc_send_time_ for non-t2 cases as well. Please check. 

This variable is not passed to the _pfc_storm_sonic.j2_ and _pfc_storm_stop_sonic.j2_ template files for non-t2 cases. We specifically pass this variable only for T2 templates alone. We don't have non-t2 topology test beds to test non-t2 cases.